frontend/components/TableContainer/DataTable/LinkCell/LinkCell.tsx

```diff
@@ -15,7 +15,7 @@ const LinkCell = ({
   value,
   path,
   title,
-  classes = "w250",
```

Comment:
Removing the default value as it was applying in areas we didn't want it. instead, rely on parent to pass the classes they want to use.

Reply:
@lukeheath we'll probably want to do a pass of all the tables using lengthy values for text fields and link fields to be sure that removing this default works with the rest of the responsive table refactor. I don't remember specifically where but I believe there were at least a couple places in the refactor where I was relying on the default rather than passing in a value from the parent. Is this something you think we can address as part of QA?